implementations/rust/ockam/ockam_command/src/command_global_opts.rs

```diff
@@ -210,24 +211,25 @@ impl CommandGlobalOpts {
     }
 }
 
-#[cfg(test)]
-impl CommandGlobalOpts {
-    pub fn new_for_test(global_args: GlobalArgs, state: CliState) -> Self {
-        let terminal = Terminal::new(
-            global_args.quiet,
-            global_args.no_color,
-            global_args.no_input,
-            global_args.output_format.clone(),
-        );
-        Self {
-            global_args,
-            state,
-            terminal,
-            rt: Arc::new(Runtime::new().expect("cannot initialize the tokio runtime")),
-            tracing_guard: None,
-        }
-    }
-}
+// #[cfg(test)]
```

Comment:
Remove that code?

Reply:
I commented to check if the compiler was happy and completely forgot about it.